app/scripts/controllers/eventRegistrations.js

```diff
@@ -386,7 +386,15 @@ angular.module('confRegistrationWebApp')
       }
 
       //update registration
-      $http.put('registrations/' + registrant.registrationId, $scope.getRegistration(registrant.registrationId)).error(function(){
+      var registrationIndex = _.findIndex($scope.registrations, { 'id': registrant.registrationId });
+      var registrantIndex = _.findIndex($scope.registrations[registrationIndex].registrants, { 'id': registrant.id });
+      $scope.registrations[registrationIndex].registrants[registrantIndex] = registrant;
```

Comment:
I thought it did tool.  But we had some users report that withdraws and checkins not saving.  I think it might be related to this.  When the registration object is updated (with a GET request).  I don't think it is propagating down properly.